kotlinx-coroutines-core/jvm/src/Executors.kt

```diff
@@ -38,13 +38,23 @@ public abstract class ExecutorCoroutineDispatcher: CoroutineDispatcher(), Closea
 
 /**
  * Converts an instance of [ExecutorService] to an implementation of [ExecutorCoroutineDispatcher].
+ *
+ * Note, that if the underlying executor uses limited queues and throws [RejectedExecutionException] on
+ * attempt to submit a task, then the [Job] of the affected task is [cancelled][Job.cancel] and the
```

Comment:
I don't know if this is the right point to explain that a cancellation can happen at coroutine start, cancellation and in any suspension point.

Reply:
That's probably going too much into the details and basics that are explained elsewhere.